docs/tutorials/extend_mapping_pipeline_with_digital_twin.md

```diff
@@ -0,0 +1,72 @@
+# Extend your pipeline with a Digital Twin Service
+
+So far we have been consuming data directly from Vorto Middleware and using it in our GeoLocation App. In the absence of a persistence layer there is no way to know the last active location of the assets. So lets extend our pipeline with a digital twin service.
+
+In simple words digital twin is the virtual representation of a physical asset. All major cloud vendors provide a digital twin service and minimally all of them have a feature of persisting the last known state of the device. We will be using Eclipse Ditto aka Bosch IoT Things as the digital twin service for this tutorial.
+
+The digital twin service will be placed between the Vorto Middleware and the IoT Solution.
```

Comment:
I would suggest to explain the picture a bit. For example
"The Digital Twin Service receives the converted device data from the Eclipse Vorto Middleware and persists it. An IoT Solution can use the Bosch IoT Things REST API to fetch the data. "